tests/Directory.Build.props

```diff
@@ -5,7 +5,7 @@
     <PropertyGroup>
         <DeveloperBuildTestTfms>netcoreapp3.0</DeveloperBuildTestTfms>
         <StandardTestTfms>$(DeveloperBuildTestTfms)</StandardTestTfms>
-        <StandardTestTfms Condition="'$(OS)' == 'Windows_NT'">$(StandardTestTfms);net48</StandardTestTfms>
+        <StandardTestTfms Condition="'$(OS)' == 'Windows_NT'">$(StandardTestTfms);net48;net45</StandardTestTfms>
```

Comment:
The tests for `net48` will pick `net45` and not `netstandard2.0`.
So there are no (explicit) tests for `netstandard2.0`, though the code-paths for `net45` and `netstandard2.0` are the same.

So either add tests for `netstandard2.0` or remove `net45` from the tests.